tools/profileBuilder/model/transforms.go

```diff
@@ -0,0 +1,313 @@
+// Package model holds the business logic for the operations made available by
+// profileBuilder.
+//
+// This package is not governed by the SemVer associated with the rest of the
+// Azure-SDK-for-Go.
+package model
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/printer"
+	"go/token"
+	"log"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/marstr/collection"
+	goalias "github.com/marstr/goalias/model"
+	"golang.org/x/tools/imports"
+)
+
+type Alias struct {
+	*goalias.AliasPackage
+	TargetPath string
+}
+
+const armPathModifier = "mgmt"
+
+var packageName = regexp.MustCompile(`services[/\\](?P<provider>[\w\-\.\d_\\/]+)[/\\](?:(?P<arm>` + armPathModifier + `)[/\\])?(?P<version>v?\d{4}-\d{2}-\d{2}[\w\d\.\-]*|v?\d+\.\d+[\.\d\w\-]*)[/\\](?P<group>[/\\\w\d\-\._]+)`)
+
+// BuildProfile takes a list of packages and creates a profile

```

Comment:
I presume `transforms` and `transforms_test` are mostly a copy paste from `program` and `program_test` would have been good if we can keep the history  ( i.e. move and rename the file -> commit  -> modify -> commit) .

Reply:
Fixed.